### ISSUE TITLE
Prolonged P wave

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7598]
**Preferred term label:**
Prolonged P wave

**Synonyms**
Prolonged P wave duration

**Definition (free text, please give PubMed ID)**
Prolonged P wave duration is a marker of delayed inter-atrial conduction
PMID:25425321

Prolonged P wave duration has been associated with atrial fibrillation and with increased all-cause, cardiovascular disease and stroke mortality. Various cut off values have been suggested for prolonged P wave duration, generally between 110 ms or 120 ms.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614049 

PMID:23348765  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
613120 

PMID: 21051419  1/3

**Comment by pnrobinson** [comment]
added
